R/dash.R

```diff
@@ -703,7 +737,7 @@ Dash <- R6::R6Class(
             # if any filetypes other than CSS are encountered in those which
             # are modified or deleted, restart the server
             
-            hard_reload <- updated_root || (has_assets && other_changed || other_added)
+            hard_reload <- updated_root || (has_assets && other_changed || other_added || other_deleted)
```

Comment:
The implementation has been modified such that deleting a file (which is not `*.css`) triggers a full page reload also.